lib/datadog/appsec/configuration/settings.rb

```diff
@@ -94,7 +94,7 @@ def duration(base = :ns, type = :integer)
 
         ENVS = {
           'DD_APPSEC_ENABLED' => [:enabled, Settings.boolean],
-          'DD_APPSEC_RULESET' => [:ruleset, Settings.string],
+          'DD_APPSEC_RULES' => [:ruleset, Settings.string],
```

Comment:
I wondered about that. The reason the env var is `_RULES` is because every other lib uses that but better terminology such as `ruleset` was only introduced later on (notably to distinguish between a "rule file" - which is a container - and a ruleset - singular, a set of rules, whatever its container is), and is now used throughout the code (including in `libddwaf`).

I punted to not changing it under the premise of consistency with the code + the risk of having to move back to `ruleset` again later down the road and break the public API (unless we add an alias). Sadly aforementioned terminology is not properly defined yet so I'm wary to make a call here. WDYT?

Reply:
I see. I guess I would still go for `rules`, and pre-1.0 seems like a good opportunity to change it, but I don't feel very strongly about it :)